lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -197,28 +198,17 @@ def configure(conf)
         raise Fluent::ConfigError, "host placeholder and template installation are exclusive features."
       end
 
-      if !Fluent::Engine.dry_run_mode
-        if @template_name && @template_file
-          retry_operate(@max_retry_putting_template, @fail_on_putting_template_retry_exceed) do
-            if @rollover_index
-              raise Fluent::ConfigError, "'deflector_alias' must be provided if 'rollover_index' is set true ." if not @deflector_alias
-            end
-            if @enable_ilm
-              raise Fluent::ConfigError, "'rollover_index' and 'deflector_alias' must be provided if 'enable_ilm' is set true ." if !@deflector_alias &&!@deflector_alias
-            end
-
-            verify_ilm_working if @enable_ilm
-            if @customize_template
-              template_custom_install(@template_name, @template_file, @template_overwrite, @customize_template, @enable_ilm, @deflector_alias, @ilm_policy_id)
-            else
-              template_install(@template_name, @template_file, @template_overwrite, @enable_ilm, @deflector_alias, @ilm_policy_id)
-            end
-            create_rollover_alias(@index_prefix, @rollover_index, @deflector_alias, @application_name, @index_date_pattern, @enable_ilm, @ilm_policy_id, @ilm_policy)
-          end
-        elsif @templates
-          retry_operate(@max_retry_putting_template, @fail_on_putting_template_retry_exceed) do
-            templates_hash_install(@templates, @template_overwrite)
-          end
+      if @template_name && @template_file
```

Comment:
Looks reasonable.
But, could you use `!Fluent::Engine.dry_run_mode` for using network code?
Fluentd `#configure` is run twice as dry run mode and normal mode.
I don't want to change previous behavior.

Reply:
Will fix, I forgot this conditional when moving the code.

Reply:
I've fixed test failures and this comment fix.
Could you confirm them?